src/instructlab/model/evaluate.py

```diff
@@ -275,7 +275,7 @@ def get_gpus(ctx, gpus=None) -> tuple[int | None, int]:
     # First Party
     from instructlab.model.backends.vllm import get_argument
 
-    eval_serve = ctx.obj.config.serve
+    # eval_serve = ctx.obj.config.serve
```

Comment:
can we remove this line if we dont need it rather than leave it commented?

Reply:
removed